### PR TITLE
Add group integration into user app #162

### PR DESCRIPTION
**Pull request by ndufreche**
This a group CRUD integration into user app

### CODE REVIEW

File: src/tb/apps/user/views/group/form.view.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    }
+                );
+            },
+
+            print: function (tpl) {
+                jQuery(this.selector).html(tpl);
+            },
+
+            display: function () {
+                this.dfd = jQuery.Deferred();
```
Comment (fkroockmann) [inline src/tb/apps/user/views/group/form.view.js]: Empty function ?

File: src/tb/apps/user/views/user/form.view.js [hunk 2]
```diff
@@ -45,15 +45,6 @@ define(
...
-                if (!data.hasOwnProperty('login') || data.login.trim().length === 0) {
-                    form.addError('login', 'login is required');
-                }
-                if (!data.hasOwnProperty('password') || data.password.trim().length === 0) {
-                    form.addError('password', 'password is required');
-                }
-            },
-
             display: function () {
                 this.dfd = jQuery.Deferred();
```
Comment (fkroockmann) [inline src/tb/apps/user/views/user/form.view.js]: Empty function ?

**Comment by fkroockmann** [comment]
:+1:

File: src/tb/apps/user/views/group/form.view.js (hunk 1 above)
Comment (ndufreche) [inline src/tb/apps/user/views/group/form.view.js]: Is the display style like in the user form but this time i don't use popin then I have nothing to print, the print is do in the promise of the formbuilder.

File: src/tb/apps/user/views/user/form.view.js (hunk 2 above)
Comment (ndufreche) [inline src/tb/apps/user/views/user/form.view.js]: Nop this function create a promise and display the popin.

**Pull request merged**